From eb059d08e8c6800198066e374df7254fbb750144 Mon Sep 17 00:00:00 2001
From: Liang Chen <cl@rock-chips.com>
Date: Thu, 1 Nov 2018 10:14:01 +0800
Subject: [PATCH] arm: dts: rk3128-fireprime: remove custom cma region@88000000

The address@88000000 is out of memory on rk3128-fireprime, so remove
the cma region@88000000, just delete the property <reg>, so that
system will alloc cma region automatically.

Change-Id: I2b9fdf1cd19d9fcecd59421fd551d709f9054cae
Signed-off-by: Liang Chen <cl@rock-chips.com>
---
 arch/arm/boot/dts/rk3128-fireprime.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/rk3128-fireprime.dts b/arch/arm/boot/dts/rk3128-fireprime.dts
index af0f5b28da4a..a75d814f91a7 100644
--- a/arch/arm/boot/dts/rk3128-fireprime.dts
+++ b/arch/arm/boot/dts/rk3128-fireprime.dts
@@ -114,6 +114,10 @@
 	};
 };
 
+&cma_region {
+	/delete-property/ reg;
+};
+
 &cpu0 {
 	cpu-supply = <&vdd_arm>;
 };
-- 
2.35.3

